.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -1,72 +1,82 @@
+# The azure pipeline template for Official build, and upgrade version build
+
 parameters:
 - name: 'jobFilters'
   type: object
   default: ''
 - name: 'buildOptions'
   type: string
   default: 'SONIC_CONFIG_BUILD_JOBS=1'
-- name: 'buildSlave'
-  type: string
-  default: 'n'
+- name: 'preSteps'
+  type: stepList
+  default: []
 - name: 'postSteps'
   type: stepList
   default: []
 
 jobs:
-- template: azure-pipelines-job-groups.yml
+- template: new-build-template.yml
```

Comment:
>new [](start = 12, length = 3)

`new` is not a good filename.

Reply:
fixed

Reply:
fixed

---
In reply to: [605189097](https://github.com/Azure/sonic-buildimage/pull/7198#discussion_r605189097) [](ancestors = 605189097)